marklogic-data-hub-central/ui/e2e/cypress/integration/explore/verifyDataInDatabases.spec.tsx

```diff
@@ -73,7 +73,7 @@ describe("Verify All Data for final/staging databases and non-entity detail page
     browsePage.waitForSpinnerToDisappear();
     cy.waitForAsyncRequest();
     browsePage.getSelectedEntity().should("contain", "Customer");
-    browsePage.getTotalDocuments().should("be.equal", 10);
+    browsePage.getTotalDocuments().should("be.gte", 10);
```

Comment:
Why is there uncertainty here? gte instead of equal?

Reply:
Thanks for the comment, changed it to 11.